packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/ViewManagerDefinition.kt

```diff
@@ -23,6 +23,8 @@ class ViewManagerDefinition(
 
   fun createView(context: Context): View = viewFactory(context)
 
+  fun getPropsNames(): List<String> = props.keys.toList()
```

Comment:
Up to you, in Kotlin generally there's a convention to use properties over functions:
```suggestion
  val propsNames: List<String>
    get() = props.keys.toList()
```